@here/harp-omv-datasource/lib/OmvUtils.ts

```diff
@@ -22,8 +22,9 @@ export function isArrayBufferLike(data: any): data is ArrayBufferLike {
 /**
  * @hidden
  */
-export function lat2tile(lat: number, zoom: number): number {
-    return Math.floor(
+export function lat2tile(lat: number, zoom: number, round: boolean = false): number {
+    const func = round ? Math.round : Math.floor;
```

Comment:
maybe instead of passing a boolean you can just change the signature of the function and pass Math.round and Math.floor directly.

export function lat2tile(lat: number, zoom: number, converter = Math.round)  {
 return converter(...)
}

Reply:
Done.